### PR TITLE
NE-367: Add logLevel for CoreDNS

### PR DESCRIPTION
**Pull request by miheer**
Add logLevel for CoreDNS.

### CODE REVIEW

**Comment by miheer** [comment]
@Miciah  @alebedev87  @arjunrn  can you please review this PR?

**Comment by miheer** [comment]
Still work in progress.

File: pkg/operator/controller/names.go [hunk 1]
```diff
@@ -28,6 +28,12 @@ const (
 
 	// DefaultDNSName is the default name of dns resource.
 	DefaultDNSName = "default"
+
+	DefaultOperatorNamespace = "openshift-dns-operator"
+
+	ContainerNameOfDNSOperator = "dns-operator"
+
+	DeploymentNameOfIngressOperator = "dns-operator"
```
Comment (rfredette) [inline pkg/operator/controller/names.go:36]: Should this be `DeploymentNameOfDNSOperator`?

**Comment by rfredette** [comment]
Just the one nit, otherwise it all looks good to me

**Pull request merged**